### ISSUE TITLE
Delete in article tries to redirect to deleted article

### ISSUE THREAD

**Issue by xsteadfastx** [wallabag/wallabag#1964]
### Issue details

When i want to delete a article in the article view i get a 404 error because it reloads the article itself which is not there anymore.

```
wallabag_1 | ::ffff:172.18.0.7 - - [22/Apr/2016:07:17:16 +0000] "GET /annotations/136?entry=136 HTTP/1.0" 200 21 "https://wallabag.foo.bar/view/136" "Mozilla/5.0 (X11; Ubuntu; Linux i686; rv:43.0) Gecko/20100101 Firefox/43.0"
wallabag_1 | ::ffff:172.18.0.7 - - [22/Apr/2016:07:17:24 +0000] "GET /delete/136 HTTP/1.0" 302 408 "https://wallabag.foo.bar/view/136" "Mozilla/5.0 (X11; Ubuntu; Linux i686; rv:43.0) Gecko/20100101 Firefox/43.0"
wallabag_1 | ::ffff:172.18.0.7 - - [22/Apr/2016:07:17:24 +0000] "GET /view/136 HTTP/1.0" 404 471 "https://wallabag.foo.bar/view/136" "Mozilla/5.0 (X11; Ubuntu; Linux i686; rv:43.0) Gecko/20100101 Firefox/43.0"
```

### Environment

* wallabag version 2.0.2

**Comment by nicosomb** [comment]
Are you sure you are in 2.0.2 ? 
I just tried and it seems working.

**Comment by xsteadfastx** [comment]
yes i am. tried it again... it always ends with a 404. im using the
nginx and the config from the docs. mh. somehow the redirect doesnt
work.

**Comment by nicosomb** [comment]
Can you try to debug please? 
In EntryController.php, line 415, you have:

`$to = ($url !== $request->headers->get('referer') ? $request->headers->get('referer') : null);`

Can you add: 

`var_dump($request->headers->get('referer'));die;`

And try to delete an other article please?

**Comment by xsteadfastx** [comment]
`string(40) "https://wallabag.foo.bar/view/66" `

**Comment by nicosomb** [comment]
And can you replace the new line by
`var_dump($request->headers->get('referer'));die;` please?

**Comment by xsteadfastx** [comment]
`string(24) "http://wallabag/view/169" `. ok there is the domain missing.

**Comment by bmillemathias** [comment]
I have the very same problem, I delete an article and I get a 404 error. I use nginx as reverse proxy which route to my lxc machine which have nginx + php7 with the standard vhost configuration for wallabag as provided in the documentation.

I can see such messages in ``var/prod.log`` inside the project directory
```
[2016-05-08 19:08:01] request.INFO: Matched route "view". {"route_parameters":{"_controller":"Wallabag\\CoreBundle\\Controller\\EntryController::viewAction","id":"2","_route":"view"},"request_uri":"http://wallabag.mydomain.fr/view/2"} []
[2016-05-08 19:08:01] security.DEBUG: Read existing security token from the session. {"key":"_security_secured_area"} []
[2016-05-08 19:08:01] security.DEBUG: User was reloaded from a user provider. {"username":"baptistemm","provider":"Symfony\\Bridge\\Doctrine\\Security\\User\\EntityUserProvider"} []
[2016-05-08 19:08:01] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "Wallabag\CoreBundle\Entity\Entry object not found." at /srv/www/wallabag/var/cache/prod/classes.php line 2093 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): Wallabag\\CoreBundle\\Entity\\Entry object not found. at /srv/www/wallabag/var/cache/prod/classes.php:2093)"} []
[2016-05-08 19:08:01] security.DEBUG: Stored the security token in the session. {"key":"_security_secured_area"} []
```

**Comment by steckerhalter** [comment]
I have this same issue. Tried with self-contained and now with manually installed 2.0.5. When I go to `https://example.com/archive/1` it gets into a redirect loop always archiving and unarchiving. I'm using Apache 2.4

I tried using a vhost like described on http://doc.wallabag.org/en/master/user/installation.html#virtual-hosts and also with `AllowOverride all` and no other directives (which then should use the .htaccess, right?
